tensorflow/examples/label_image_go/.gitignore

```diff
@@ -0,0 +1,2 @@
+data/imagenet_comp_graph_label_strings.txt
```

Comment:
This content is downloaded using wget [here](https://github.com/tensorflow/tensorflow/pull/1771/files/a6e723f10b171f649213ce400b75c63e72a8912d#diff-4bf5580e9acb09fd4dd93385e59673a5R18)
I added it in the .gitignore because I uploaded it a pair of times to the repo :)